docs/guide/solr.md

```diff
@@ -300,13 +299,82 @@ ez_search_engine_solr:
 The configuration above will result in the following boosting (Content Type / Field):
 
 - `article/title: 2.0`
-- `news/title: 3.0`
-- `blog_post/title: 1.5`
 - `news/description: 1.0` (default)
 - `article/text (meta): 5.0`
 - `blog_post/name (meta): 10.0`
 - `article/name (meta): 2.0`
 
+!!! tip "How to configure boosting on specific fields"
+
+    Currently, boosting on particular fields is missing. However, it could be configured using 3rd party [Novactive/NovaeZSolrSearchExtraBundle](https://github.com/Novactive/NovaeZSolrSearchExtraBundle) in case of custom search implementation, for instance for your front-end search form. Unfortunately, this doesn't affect search perfomed in PlatformUI. 
```

Comment:
> for instance for your front-end search form
Two "for's" in 3 words, maybe it would be better to rewrite this part to something like:
> for instance to handle your front-end search form

Reply:
Hmm, right. @MagdalenaZuba offered to fix some language mistakes, so I hope she'll cover also this one.